### ISSUE TITLE
Installation issues - HTTP error 401

### ISSUE THREAD

**Issue by valentinitnelav** [valentinitnelav/plotbiomes#8]
```
> devtools::install_github("valentinitnelav/plotbiomes")
Using github PAT from envvar GITHUB_TOKEN
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 401.
  Bad credentials

  Rate limit remaining: 56/60
  Rate limit reset at: 2021-10-06 23:56:59 UTC

```
Anything to be done other than cloning?

_Originally posted by @technocrat in https://github.com/valentinitnelav/plotbiomes/issues/7#issuecomment-937326376_

**Comment by valentinitnelav** [comment]
Hi @technocrat, I moved this as a separate issue from #7 - "Use GitHub pages for documentation", since that is a different issue. 

To add a bit more, I also tried both under Linux (connected with GitHub) and Windows (not connected with my GitHub account) and I cannot reproduce your error.

I admit that I do not really have time to maintain this package, but not sure where the problem is. I had something similar once with issue #6 - "Failed to install - unsupported proxy error message". Are you limited by some firewall / admin rights?
